.travis.yml

```diff
@@ -51,6 +51,8 @@ before_script:
 script:
   # Run tests, including PEP8 check, and produce a coverage report
   - py.test --doctest-modules --cov --cov-report term-missing --pep8 $TRAVIS_BUILD_DIR/odl
+  # Invoke also the alternative way of running the unit tests
```

Comment:
Does this cause travis to run twice? In that case i'm sceptical, that will take some time.